cfme/tests/services/test_myservice.py

```diff
@@ -123,3 +124,18 @@ def test_download_file(appliance, context, needs_firefox, myservice, filetype):
     with appliance.context.use(context):
         myservice = MyService(appliance, name=service_name, vm_name=vm_name)
         myservice.download_file(filetype)
+
+
+# Automation test for BZ1443772
+@pytest.mark.parametrize('context', [ViaUI])
+def test_service_link(appliance, context, myservice):
+    """Tests service link from VM details page"""
+
+    service_name, vm_name = myservice
+    with appliance.context.use(context):
+        myservice = MyService(appliance, name=service_name, vm_name=vm_name)
+        earlier_title = 'Service "' + service_name + '"'   # Page title before navigation
+        view = navigate_to(myservice, 'VMDetails')
+        view.relationships.click_at("Service")
```

Comment:
**REQUIRED**
I guess here the has been changed. So you need to create a new respectful view and instead asserting titles you can just assert `is_displayed` property.